llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -14476,6 +14477,60 @@ SDValue SITargetLowering::performFDivCombine(SDNode *N,
   return SDValue();
 }
 
+SDValue SITargetLowering::performFMulCombine(SDNode *N,
+                                             DAGCombinerInfo &DCI) const {
+  SelectionDAG &DAG = DCI.DAG;
+  EVT VT = N->getValueType(0);
+  EVT i32VT = VT.changeElementType(MVT::i32);
```

Comment:
```suggestion
  EVT IntVT = VT.changeElementType(MVT::i32);
```

Reply:
This change wasn't done? Variable names should start with a capital letter.